src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeArm.java

```diff
@@ -0,0 +1,35 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import java.util.Collections;
+import java.util.LinkedList;
+
+public class NodeArm {
+    private int id;
+    private final LinkedList<NodeArm> neighbors;
+    private final double anglePos;
+    private final double extendPos;
+    public NodeArm(int id, double extenderPos, double anglePos){
+        this.id = id;
+        this.extendPos = extenderPos;
+        this.anglePos = anglePos;
+        neighbors = new LinkedList<NodeArm>();
+    }
+    public double getExtendPos() {
+        return extendPos;
+    }
+    public double getAnglePos() {
+        return anglePos;
+    }
+    public void setNeighbors(NodeArm[] neighbors) {
```

Comment:
this function is not a set function since it only adds and cant remove also neighbors should not be added after creation i understand why this cant be in the constructor but perhaps make the function protected or only work once

Reply:
changed name to add and changed it to happen once